src/main/java/IO/UserModPrimitiveConverter.java

```diff
@@ -0,0 +1,71 @@
+package IO;
```

Comment:
wrong package name IO
Also pelase remove all files that are not related to csv format. I think they can be easily added back if needed.
Also pelase move usererpository to domain package and move DT object to logic package
Please rename logic package to something more meaningful like mapreduce or so.